src/app/content/hooks/locationChange.spec.ts

```diff
@@ -13,7 +13,8 @@ import { receiveBook, receivePage } from '../actions';
 import * as routes from '../routes';
 import { formatBookData } from '../utils';
 
-const mockConfig = {BOOKS: {
+// tslint:disable-next-line
+var mockConfig = {BOOKS: { // var is needed so that the mock is hoisted
```

Comment:
this wasn't broken before, why does it need to be changed?

Reply:
I have no idea 😭 without that we get error:
```
 FAIL  src/app/content/hooks/locationChange.spec.ts
  ● Test suite failed to run

    ReferenceError: mockConfig is not defined

      21 | };
      22 |
    > 23 | jest.mock('../../../config', () => mockConfig);
         |                                    ^
      24 |
      25 | describe('locationChange', () => {
      26 |   let store: Store;

      at Object.mockConfig [as user:/Users/nas/code/ox/rex-web/src/config.js:] (src/app/content/hooks/locationChange.spec.ts:23:36)
      at Object.<anonymous> (src/helpers/Sentry.ts:3:1)
      at Object.<anonymous> (src/app/errors/components/ErrorBoundary.tsx:3:1)
      at Object.<anonymous> (src/app/navigation/components/NavigationProvider.tsx:4:1)
      at Object.<anonymous> (src/app/navigation/components/index.ts:1:1)
      at Object.<anonymous> (src/app/navigation/index.ts:2:1)
      at Object.<anonymous> (src/app/errors/reducer.ts:3:1)
      at Object.<anonymous> (src/test/createTestStore.ts:6:1)
      at Object.<anonymous> (src/app/content/hooks/locationChange.spec.ts:5:1)
```